tests/bot/test_skill_orchestration.py

```
@@ -76,3 +76,112 @@ def test_skill_orchestration_re_exports_share_identity_with_bot_core():
+                sa.select(Path).where(Path.domain == "dataset")
+            )
+        ).scalars().all()
+
+    namespaces = {r.namespace for r in rows}
+    paths = {r.path for r in rows}
+
+    assert "telegram/alice" in namespaces, (
+        f"auto-capture did not land in session namespace; got namespaces={namespaces}"
+    )
+    assert "__shared__" not in namespaces, (
+        f"auto-capture leaked into __shared__ — D5 silent fallback regressed"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)